src/Dice.elm

```diff
@@ -44,46 +44,46 @@ rollD faces seed =
 roll2D : Int -> Seed -> ( ( Int, Int ), Seed )
 roll2D faces seed =
     let
-        ( a, seed' ) =
+        ( a, seed_ ) =
             rollD faces seed
 
-        ( b, seed'' ) =
-            rollD faces seed'
+        ( b, seed__ ) =
+            rollD faces seed_
     in
-        ( ( a, b ), seed'' )
+        ( ( a, b ), seed__ )
 
 
 roll3D : Int -> Seed -> ( ( Int, Int, Int ), Seed )
 roll3D faces seed =
     let
-        ( ( a, b ), seed' ) =
+        ( ( a, b ), seed_ ) =
```

Comment:
lol... it's pretty funny that we're going to see a lot of this kind of thing from now on... it's pretty much the same thing isn't it? I don't really understand why they wanted to get rid of the `'` (prime).